lib/matplotlib/__init__.py

```diff
@@ -1525,261 +1534,152 @@ def _replacer(data, key):
 """
 
 
-def _add_data_doc(docstring, replace_names, replace_all_args):
+def _add_data_doc(docstring, replace_names):
     """Add documentation for a *data* field to the given docstring.
 
     Parameters
     ----------
     docstring : str
         The input docstring.
-    replace_names : list of strings or None
+    replace_names : List[str] or None
```

Comment:
According to numpydoc this shoud be `list of str`. I don't want to start a big discussion about this because it's a private function anyway, but we've been using `list of ...` everywhere so far.

Reply:
I think standard type annotations actually work quite well in cases like this but sure, let's decide this in some separate discussion.